### PR TITLE
[Merged by Bors] - cli: Add `fluvio cluster status`

### PR DESCRIPTION
**Pull request by crajcan**
### Behavior

Adds `fluvio cluster status` to the CLI

Tries to connect to the SC, reports "none" if unsuccessful, describes where (local/k8s/cloud) cluster is deployed if successful. If the SC is up, it then will report if the SPUs are down, or if they are all empty.

### Comment

I'm not sure if this was still wanted but it looked like a fun first issue. It's also possible I misinterpreted the original request. Specifically, I'm not sure how we could determine if the status should be "stopped (no sc, spu running but data exits)", as I'm not sure how to determine if there is data in the spus if the connection to the sc fails.

I would love some feedback and would be happy to make modifications!

https://github.com/infinyon/fluvio/issues/1673

### CODE REVIEW

**Comment by sehz** [comment]
Running changes show following: 
```
$  flvd cluster status
Cluster Running on local k8s
spus have no data
```

**Comment by sehz** [comment]
The response should model something like `fluvio cluster start.`  So something like this would be good:

```
$ fluvio cluster status
📝 Running checks using profile 'rancher-desktop`
    ✅ Kubectl active cluster rancher-desktop at: https://127.0.0.1:6443 found
    ✅ SC is ok
    ✅ 2/2 SPU is ok
    ✅ 10 topics with 25G used
```

I will update original ticket as well

**Review by sehz — CHANGES_REQUESTED** [review]
Nice start.  Updated with better specification

File: crates/fluvio-cluster/src/cli/status.rs [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            if let Some(storage) = topic.spec.get_storage() {
+                partition_size = storage.max_partition_size.unwrap();
+            }
+
+            replications * partitions * partition_size
+        }).sum()
+    }
+
+    fn human_readable_size(size: u64) -> String {
+        if size < 1024 {
```
Comment (sehz) [inline crates/fluvio-cluster/src/cli/status.rs:224]: can you change to use https://crates.io/crates/bytesize?

**Comment by crajcan** [comment]
> LGTMS. Thanks for contribution!

😄 Thanks for the reviews!

**Comment by sehz** [comment]
clippy failures

**Pull request closed without merging**